db/migrate/20210208145233_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
we don't want this column to be `null`. it is either `true` or `false`
```suggestion
    add_column :users, :admin, :boolean, null: false, default: false
```

Reply:
OK :+1:  Muss ich noch ein Migration dafür machen, wenn ich diese column editiere?

Reply:
Du kannst die Migration solange bearbeiten, wie der PR noch nicht gemerged ist. Du bist die einzige, die das auf ihrem Rechner schon ausgeführt hat, darum ist das erlaubt. Allerdings musst Du Dein Schema bei Dir lokal in Ordnung bringen. Das sollte mir db:rollback in diesem Fall kein Problem sein. db:rollback macht die jeweils letzte Migration rückgängig. Du kannst es auch mehrmals ausführen, um mehrere Schritte zurückzugehen. Allerdings gibt es Schritte, die nicht rückgängig gemacht werden können.
Ich finde es im Prinzip besser, nicht so viele Migrationen zu haben, sondern es mögichst übersichtlich zu halten. Manchmal ist es aber kompliziert und ich lege auch einfach eine neue an. In diesem Fall ist es sowieso nicht wichtig und Dir überlassen, ob Du das mit rollback mal ausprobieren willst.

Reply:
OK, ich probiere es mit ein Rollback aus :)